### PR TITLE
lid: add Spanish translation

### PR DESCRIPTION
**Pull request by kant**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.es/linux/lid.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# lid
+
+> NOTA: Esta página es actualmente una de redirección. Si está familiarizado con este programa, por favor abra una solicitud de extracción.
+> Consulta la base de datos ID y reporta los resultados.
+> En Fedora y Arch Linux, el nombre binario `lid` es tomado por otro programa. Ver `tldr libuser-lid`.
+> Más información: <https://www.gnu.org/software/idutils/>.
+
+- Ver documentación de `libuser-lid`:
```
Comment (tricantivu) [inline pages.es/linux/lid.md:8]:
```suggestion
- Ve documentación de `libuser-lid`:
```

**Review by tricantivu — CHANGES_REQUESTED** [review]

**Comment by tricantivu** [comment]
LGTM, thanks.

File: pages.es/linux/lid.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# lid
+
+> NOTA: Esta página es actualmente una de redirección. Si está familiarizado con este programa, por favor abra una solicitud de extracción.
+> Consulta la base de datos ID y reporta los resultados.
+> En Fedora y Arch Linux, el nombre binario `lid` es tomado por otro programa. Ver `tldr libuser-lid`.
```
Comment (tricantivu) [inline pages.es/linux/lid.md:5]:
```suggestion
> En Fedora y Arch Linux, `lid` es otro programa. Vea `tldr libuser-lid`.
```

File: pages.es/linux/lid.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# lid
+
+> NOTA: Esta página es actualmente una de redirección. Si está familiarizado con este programa, por favor abra una solicitud de extracción.
+> Consulta la base de datos ID y reporta los resultados.
```
Comment (tricantivu) [inline pages.es/linux/lid.md:4]:
```suggestion
> Consulta la base de datos de identificadores y reporta los resultados.
```

**Review by tricantivu — DISMISSED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**